### PR TITLE
wallpaper v2.0.0

### PR DESCRIPTION
**Pull request by stek29**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

It was rewritten from scratch, and pre-sierra support was dropped.
So, maybe it makes sense to create a different formula for new version?

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> swift build -c release -Xswiftc -static-stdlib --disable-sandbox
error: package at '/private/tmp/wallpaper-20180930-54698-t8uy4h/macos-wallpaper-2.0.0' requires a minimum Swift tools version of 4.2.0 (currently 4.0.0)
```

**Comment by fxcoudert** [comment]
We have a PR open for swift 4.2, but it is failing to build: https://github.com/Homebrew/homebrew-core/pull/32223
Any help there is welcome

File: Formula/wallpaper.rb [hunk 1]
```diff
@@ -14,12 +14,15 @@ class Wallpaper < Formula
     sha256 "7a715b58b8f4e654b409347fd8cff2bce6a3dfc83b0d345c3b83fc223eaf952a" => :yosemite
   end
 
+  depends_on :xcode => ["9.0", :build]
```
Comment (sindresorhus) [inline Formula/wallpaper.rb]: Should use Xcode 10 to build.

File: Formula/wallpaper.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Wallpaper < Formula
   desc "Get or set the desktop wallpaper"
```
Comment (sindresorhus) [inline Formula/wallpaper.rb]: Can you update it to:

> Manage the desktop wallpaper

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**